scanpipe/pipes/d2d.py

```diff
@@ -421,7 +421,10 @@ def _map_path_resource(
     # Only create relations when the number of matches if inferior or equal to
     # the current number of path segment matched.
     if len(match.resource_ids) > match.matched_path_length:
-        to_resource.update(status=flag.TOO_MANY_MAPS)
+        # Single matched path segment/file name don't
```

Comment:
IMHO we should go further and not store any mapping at all in this case

Reply:
Ack, So we don't want to path map any files solely based on the file name.